foundations/foundation-vertx/src/main/java/io/servicecomb/foundation/vertx/http/FileUploadPart.java

```diff
@@ -0,0 +1,65 @@
+/*

```

Comment:
Please update the license file template.

Reply:
sorry, did not update code template at home

Reply:
done